ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -191,6 +191,9 @@ class MetricProbeFactory(ProbeFactory):
 
     @classmethod
     def update_args(cls, args, attribs):
+        # adding probe_id to probe-tags so it would be recorded as a metric tag
+        args["tags"]["debugger.probeid"] = args["probe_id"]
```

Comment:
```suggestion
        args["tags"]["debugger.probeId"] = args["probe_id"]
```
?

Reply:
try to duplicated other impls:
dotnet: https://github.com/DataDog/dd-trace-dotnet/blob/0b41807e62195425478a3831773b7d15e25eca46/tracer/src/Datadog.Trace/Debugger/Instrumentation/SpanDebuggerInvoker.cs#L46
java: https://github.com/DataDog/dd-trace-java/blob/848d3775b0c6d1ce8cda07fe080f64f57e782d97/dd-java-agent/agent-debugger/src/main/java/com/datadog/debugger/instrumentation/Instrumentor.java#L31